nixos/modules/system/boot/stage-1-init.sh

```diff
@@ -119,6 +119,14 @@ specialMount() {
 }
 source @earlyMountScript@
 
+# Copy initrd secrets staged into /initrd-secrets to their real destinations
+if [ -d "/initrd-secrets" ]; then
+    for secret in $(cd "/initrd-secrets"; find . -type f); do
+        mkdir -p $(dirname "/$secret")
+	cp "/initrd-secrets/$secret" "$secret"
```

Comment:
```suggestion
    cp "/initrd-secrets/$secret" "$secret"
```

Reply:
This has been addressed, resolving conversation.